features/api/ideas_workshop_ideas.feature

```diff
@@ -823,17 +823,12 @@ Feature:
   Scenario: As a logged-in user I can not publish an idea that has another status than PENDING at the moment of execution
```

Comment:
Please, change this scenario description: only idea in the DRAFT status can be published.

Reply:
done tnx